### DISCUSSION TITLE
Is tere lookahead equivalent pattern that I could use as it is not supported in regex?

### DISCUSSION THREAD

**Discussion by DanielMazurkiewicz** [tree-sitter/tree-sitter#2100] [Q&A]
Trying to match just content of comment:
```
/*
  some comment
*/
```
so ideally it would look more less like that:
```
seq(
   "/*",
  token(__match_everything_till__("*/"))
   "*/"
)
```

Can I achieve that somehow?

**Comment by RedCMD** [comment]
There is no lookahead that I no of
the best you can do is something like this: `token(seq("/*", /[^*]*\*+([^/*][^*]*\*+)*/, "/"))`
http://stackoverflow.com/questions/13014947/regex-to-match-a-c-style-multiline-comment/36328890#36328890

sadly it has very little structure
but does work

**Comment by matthias-Q** [comment]
I have a similar issue: I want to match everything inside triple quotes like:
```
"""
a,b,c
1,2,3
4,5,6
"""
```

```
"""{
    "columns": ["a", "b", "c"],
    "data": [
        [1, "x", false],
        [4, "y", null]
    ]
}"""

```

I have tried:
```javascript
        _inner_triple_quotes: _ => /[^\"{3}]*[^\"{3}]/,
        _triple_quote_string: $ => seq('"""', $._inner_triple_quotes, '"""'),
```
Any ideas?

**Reply by RedCMD** [reply]
> _inner_triple_quotes: _ => /[^\"{3}]*[^\"{3}]/

@matthias-Q `{3}` is not supported inside character classes?
`[^\"{3}]` would not match `"`, `{`, `3` and `}`

**Reply by matthias-Q** [reply]
Thanks, I solved it using the other suggestion in #2094

**Comment by ahlinc** [comment]
Look to another discussion and a https://github.com/tree-sitter/tree-sitter/discussions/2094#discussioncomment-5082953
